src/emulator/controller.ts

```diff
@@ -374,6 +375,52 @@ export async function startAll(options: any, noUi: boolean = false): Promise<voi
 
     const databaseEmulator = new DatabaseEmulator(args);
     await startEmulator(databaseEmulator);
+
+    if (exportMetadata.database) {
+      const importDirAbsPath = path.resolve(options.import);
+      const databaseExportDir = path.join(importDirAbsPath, exportMetadata.database.path);
+
+      const files = fs.readdirSync(databaseExportDir);
+      for (const f of files) {
+        const fPath = path.join(databaseExportDir, f);
+        const ns = f.split(".json")[0];
+
+        databaseLogger.logLabeled("BULLET", "database", `Importing data from ${fPath}`);
+
+        const readStream = fs.createReadStream(fPath);
+        await new Promise((resolve, reject) => {
+          const req = http.request(
+            {
+              method: "POST",
+              host: `${databaseAddr.host}`,
+              port: databaseAddr.port,
+              path: `/.json?ns=${ns}&disableTriggers=true`,
```

Comment:
Please add [`?writeSizeLimit=unlimited`](https://firebase.google.com/docs/reference/rest/database#section-param-writesizelimit) just in case.